src/main/java/dev/j3fftw/litexpansion/machine/MassFabricator.java

```diff
@@ -35,7 +35,8 @@
         new NamespacedKey(LiteXpansion.getInstance(), "mass_fabricator"), Items.MASS_FABRICATOR_MACHINE
     );
 
-    private static final int ENERGY_CONSUMPTION = Utils.euToJ(166_666 / 100);
+    public static int ENERGY_CONSUMPTION = Utils.euToJ(166_666 / 100);
```

Comment:
Can we just finally simplify this to 16_666

Reply:
Why did we do it like this in the first place?

Reply:
166,666 eu to J but then we also had to divide for the progress and couldn't ever be bothered making it more simple.

This was actually changed in uncommited code I have which would have made the recent 200 scrap change use the same overall energy (1.6 mil) rather than the now 3.2 mil. But, since we're mean we may as well just simplify this and not do that change.

Reply:
Okay fluffy change this to 16_666 and add 300 more to the scrap

Reply:
NotLikeThis